CMakeLists.txt

```diff
@@ -303,7 +303,7 @@ add_subdirectory(lib)
 #
 
 add_executable(firmware)
-
+set_target_properties(firmware PROPERTIES INTERPROCEDURAL_OPTIMIZATION True)
```

Comment:
/very minor/
Can we move this a bit lower in the CMakeLists? I would put it under `# generate linker map file`

Reply:
I'd put that line under `add_compile_options(-Os)` - that only for the release firmware builds. Does that sound right?